website/docs/cloud-docs/api-docs/explorer.mdx

```diff
@@ -0,0 +1,459 @@
+
+Explorer queries will time out after 60 seconds. If the desired query is timing
+out, try a simpler query that uses less filters.
+
+Requests to the explorer are subject to [rate
+limiting](/terraform/cloud-docs/api-docs#rate-limiting). Explorer will return a
+429 status code when rate limiting is active for the authenticated context.
+
+Data presented by the explorer is eventually consistent. Query results typically
+contain data that represents the current state of the system, but in some cases
+a small delay may be necessary before querying for data that has recently been
+updated.
+
+### Query Parameters
+
+This GET endpoint requires a query string that supports the following properties.
+
+| Parameter      | Description |
+| -------------- | ----------------------------------------------------------------------------- |
+| `type`         | **Required.** Must be one of the following available views: `workspaces`, `tf_versions`, `providers`, or `modules`. See the [View Types](#view-types) section for more information about each. |
+| `sort`         | Optionally sort the returned data by the specified field name. The field name must use snake case and exist within the supplied view type. The field name supplied can be prefixed with a hyphen (`-`) to perform a descending sort. |
+| `filter`       | Optionally specify one or more filters to limit the data returned. See [Filtering](#filtering) for more information. |
+| `fields`       | Optionally limit the data returned only to the specified fields. The field names supplied must use snake case and be submitted in comma-separated format. If omitted, all available fields will be returned for the requested view type. |
+| `page[number]` | Optional. If omitted, the endpoint will return the first page. |
+| `page[size]`   | Optional. If omitted, the endpoint will use a default page size. |
+
+### View Types
+
+Explorer queries each require a view, specified by the `type` query string parameter.
+There are several contextual views available for querying:
+
+| Type          | Description |
+| ------------- | ----------------------------------------------------------------------------- |
+| `workspaces`  | Information about the workspaces in the target organization and any current runs associated with them. Each row returned represents a single workspace. See [View Type: workspaces](#view-type-workspaces) for available fields. |
+| `tf_versions` | Information about the Terraform versions in use across this organization. Each row returned represents a Terraform version in use. See [View Type: tf_versions](#view-type-tf_versions) for available fields. |
+| `providers`   | Information about the Terraform providers in use across the target organization. Each row returned represents a Terraform provider in use. See [View Type: providers](#view-type-providers) for available fields. |
```

Comment:
we might need to clarify what "in use" means, perhaps the fact they need have had a run with the new providers for it to be reflected?